### DISCUSSION TITLE
glb file question

### DISCUSSION THREAD

**Discussion by SefaDedeoglu** [enable3d/enable3d#232] [Help / Questions]
Hello everyone
I am trying to create a scene in blender and import to enable3d project. But this point i got some problems. 
For example: 
A map (from internet) imported as glb to scene. There is no layer for fall preventer.
There is question. 
How to get glb files? which program i should use? 
how to add preventer layer for physics.
thank you for interested in already

**Comment by yandeu** [comment]
What is a preventer layer?

**Reply by SefaDedeoglu** [reply]
i wanted to say ground

**Reply by yandeu** [reply]
I see. Well I guess for questions about using and converting glb, you should use the three.js forum.
